### DISCUSSION TITLE
Configure replacement of non-english characters in slug generation

### DISCUSSION THREAD

**Discussion by g-gullstrand** [statamic/cms#7801] [Q&A] [answered]
When using a language other than english, special charachters in titles etc are replaced when the slug is generated.
In swedish for example, the title of "Säkerhet" would generate a slug of "saekerhet". This is ok, byt I would rather want it replace the "ä" with just "a". 

Is it possible to configure what non-english characters should be replaced by?

**Comment by joshuablum** [comment, marked as answer]
Hej @g-gullstrand,

There is no built-in solution to this. Have a look at this PR https://github.com/statamic/cms/pull/5494 for more in-depth details.

The easiest solution would be to [listen for the EntrySaving event](https://statamic.dev/extending/events#entrysaving) and manipulate the slug on-the-fly (like replacing `ä` with `a`). Should only be a few lines of code 👍

You can use a regex if you want to replace more than that one type of character. Not familiar with Swedish apart from Ikeå product names 😅

Hope that helps :)
